tests/Fixtures/Integration/priority/strict_comparison,binary_operator_spaces.test

```diff
@@ -1,13 +1,13 @@
 --TEST--
 Integration of fixers: strict_comparison,binary_operator_spaces.text.
 --RULESET--
-{"strict_comparison": true, "binary_operator_spaces": {"operators":{"===":"align"}}}
+{"strict_comparison": true, "binary_operator_spaces": {"operators":{"===":"align_single_space"}}}
 --EXPECT--
 <?php
-$a = $b === $c;
-$d = $e === $f;
+$a = $b     === $c;
+$d = $e     === $f;
```

Comment:
looks super weird. are those spaces removed by anything? if so, do we have itest for it (binary_operator_spaces + sth) ?

Reply:
By _default_ the `binary_operator_spaces` replaces the spaces to one space for both `==` and `===`, therefore to prove the prio issue I only configured fixing `===` to alignment.